MinimOsd_Extra/Panels.h

```diff
@@ -1383,7 +1383,7 @@ static void panRose(point p){
 // Serial.printf_P(PSTR("Rose buf=%s\n"), buf_show);
 
     if(has_sign(p))
-	osd.print_P(PSTR( "\x20\xc7\xc7\xc7\xc7\x2e\xc7\xc7\xc7\xc7\x20|"));
+      osd.print_P(PSTR( "\x20\x20\x20\x24\xcb\xb8\xb9\xcb\x24\x20\x20\x20|"));
```

Comment:
This makes the screen look a little better, but the work is much worse: you'll never get the exact center on an even size. On your version you can't move due north.